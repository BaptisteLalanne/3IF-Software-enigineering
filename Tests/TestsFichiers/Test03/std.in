1
2.0
44.0
46
2019-01-01
2019-01-04
0
