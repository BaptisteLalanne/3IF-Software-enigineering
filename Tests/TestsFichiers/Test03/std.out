Acces aux donnees...
Donnees ajoutees avec succes!
4 capteurs non fiables detectes

Bienvenue dans le Menu Principal !

---------------------------MENU---------------------------
Choississez parmi les propositions suivantes
1: Calculer moyenne de la qualite de l'air dans une region
0: Quitter
Votre choix : ----------------------------------------------------------

Veuillez-renseigner les informations suivantes:
Longitude : Latitude : Rayon (km) : Date de debut (AAAA-MM-JJ) : Date de fin (AAAA-MM-JJ) :
L'indice ATMO dans la region est de 6 (la zone selectionnee est recouverte par les capteurs a 40%).

---------------------------MENU---------------------------
Choississez parmi les propositions suivantes
1: Calculer moyenne de la qualite de l'air dans une region
0: Quitter
Votre choix : ----------------------------------------------------------

A bientot !
